selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindUsages.java

```diff
@@ -165,27 +147,24 @@ public void sendCommandByKeyboardInFindUsagespanel(String command) {
    * @param numLine is the number line of usage
    */
   public void selectHighlightedItemInFindUsagesByDoubleClick(int numLine) {
-    WebElement findHighlightedItem =
-        seleniumWebDriver.findElement(
-            By.xpath(format(Locators.FIND_USAGES_HIGHLIGHTED_ITEM, numLine)));
-    new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC)
-        .until(visibilityOf(findHighlightedItem));
-    findHighlightedItem.click();
-    actionsFactory.createAction(seleniumWebDriver).doubleClick(findHighlightedItem).perform();
+    seleniumWebDriverHelper.moveCursorToAndDoubleClick(
+        By.xpath(format(Locators.FIND_USAGES_HIGHLIGHTED_ITEM, numLine)));
   }
 
   /** wait a selected element in the 'find usages' panel */
   public void waitSelectedElementInFindUsagesPanel(String nameElement) {
-    new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC)
-        .until(
-            visibilityOfElementLocated(
-                By.xpath(format(Locators.FIND_USAGES_HIGHLIGHTED_ITEM, nameElement))));
-    final WebElement item =
-        seleniumWebDriver.findElement(
-            By.xpath(format(Locators.FIND_USAGES_HIGHLIGHTED_ITEM, nameElement)));
-    new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC)
+    webDriverWaitFactory
```

Comment:
You can shorten the code by using `SleniumWebDriverHelper#waitSuccessCondition(ExpectedCondition<Boolean> expression)` method.